doc/installation.rst

```diff
@@ -77,6 +77,18 @@ Install `PHIVE <https://phar.io>`_ and issue the following command:
 
     phive install php-cs-fixer # use `--global` for global install
 
+GitHub Action (Docker)
+~~~~~~~~~~~~~~~~~~
+
+To integrate php-cs-fixer as check into a GitHub Action step, you can use a configuration like this:
+
+.. code-block:: yaml
+
+    - name: PHP-CS-Fixer
+      uses: docker://ghcr.io/php-cs-fixer/php-cs-fixer:3-php8.3
```

Comment:
```suggestion
      uses: docker://ghcr.io/php-cs-fixer/php-cs-fixer:${FIXER_VERSION:-3-php8.3}
```

Reply:
I don't think that syntax is working for GitHub Action...

`uses: docker://ghcr.io/php-cs-fixer/php-cs-fixer:${{ vars.FIXER_VERSION || '3-php8.3' }}` should work here.

Reply:
I'll try !

Reply:
This syntax is not supported by GitHub Actions and will be not in the future...

See https://github.com/orgs/community/discussions/25246#discussioncomment-3247073

We need to write the version as plain text... sad.

Reply:
Would it be possible if [`jobs.<job_id>.container` was used](https://docs.github.com/en/actions/using-workflows/workflow-syntax-for-github-actions#jobsjob_idcontainer) instead of `uses`? As far as I know the latter is for running reusable workflows, not for specifying Docker image to run on. The fact it's working with `docker://` is even [undocumented](https://docs.github.com/en/actions/using-workflows/workflow-syntax-for-github-actions#jobsjob_iduses) 😅.

Reply:
The fact is we use PHP-CS-Fixer as a step of the job. 

For example : 

```yml
jobs:
  tests:
    name: Tests
    runs-on: ubuntu-latest
    steps:
      -
        name: Checkout
        uses: actions/checkout@v4
      -
        name: PHP-CS-Fixer
        uses: docker://ghcr.io/php-cs-fixer/php-cs-fixer:3-php8.3
        with:
          args: check --config=api/.php-cs-fixer.dist.php --show-progress=none -v
```

But I never used [jobs.<job_id>.container](https://docs.github.com/en/actions/using-workflows/workflow-syntax-for-github-actions#jobsjob_idcontainer), I'll try it but I don't think that is very optimized in Action context. We need to create another job only for that. Especially if we use Docker builds.

The `docker://` syntax is documented [here](https://docs.github.com/en/actions/using-workflows/workflow-syntax-for-github-actions#example-using-the-github-packages-container-registry).

Reply:
Ah, you're right, it's a step. Thank you for the feedback, and sorry for being snippy - I'm a Gitlab guy with relatively little knowledge about Github Actions, still learning 😉.